BigBaseV2/src/services/gta_data/weapon_item.cpp

```diff
@@ -0,0 +1,78 @@
+#include "weapon_item.hpp"
+#include "gta/joaat.hpp"
+
+namespace big
+{
+	weapon_item::weapon_item()
+	{
+		this->name = "";
+		this->throwable = false;
+		this->weapon_type = "";
+
+		this->hash = 0;
+		this->reward_hash = 0;
+		this->reward_ammo_hash = 0;
+	}
+	weapon_item::weapon_item(nlohmann::json& item_json)
+	{
+		this->name = item_json["Name"];
+		this->throwable = false;
+
+		if (
+			item_json.contains("TranslatedLabel") &&
+			item_json["TranslatedLabel"].contains("English") &&
+			!item_json["TranslatedLabel"]["English"].is_null()
+		) {
+			this->name = item_json["TranslatedLabel"]["English"];
+		}
+
+		if (item_json["Category"].is_null())
+		{
+			this->weapon_type = "NULL";
+		}
+		else
+		{
+			this->weapon_type = item_json["Category"];
+		}
```

Comment:
You can probably default the this->weapon_type = "NULL" and if the Category isn't null assign it.
```cpp
this->weapon_type = "NULL";
if (!item_json["Category"].is_null())
{
	this->weapon_type = item_json["Category"];
}
```